base/BaseLoader.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <section class="loader__wrap">
+    <div class="loader" />
+  </section>
+</template>
+
+<script>
+export default {
+  name: "BaseLoader"
+};
+</script>
+
+<style lang="scss">
+.loader {
+  border: 10px solid #f3f3f3;
```

Comment:
I advise working with utility CSS frameworks. You could fully rewrite this component without any custom CSS 
https://windicss.org/guide/